tools/init_env.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+ubuntu64()
+{
+        # This is used to set evnironment deps in Ubuntu-64 for compile&run x86_32/x86_64 bit Composite
+        # Tested Ubuntu version: 18.04/20.04/21.04
+        local dependencies="build-essential git cargo xorriso mtools qemu-system-i386 gcc-multilib python2.7"
+        sudo apt install ${dependencies} -y
```

Comment:
grub2 or any associated utils?

Reply:
download the rust installation script, and install cargo with it?

Reply:
OK, I will add these utils.  (I think Ubuntu 18.04 or later has grub install by default, but I will add it in the script)

Reply:
Yeah, it certainly has grub. Not sure about the utilities around grub. Doesn't hurt to have it in the list, I think.

Reply:
Fixed.